src/clients/SpokePoolClient.ts

```diff
@@ -198,9 +198,10 @@ export class SpokePoolClient {
    * @returns A list of refund requests.
    */
   public getRefundRequests(fromBlock?: number, toBlock?: number): RefundRequestWithBlock[] {
-    return fromBlock === undefined || toBlock === undefined || isNaN(fromBlock) || isNaN(toBlock)
+    const refundRequests = fromBlock === undefined || toBlock === undefined || isNaN(fromBlock) || isNaN(toBlock)
       ? this.refundRequests
       : this.refundRequests.filter((request) => request.blockNumber >= fromBlock && request.blockNumber <= toBlock);
+    return [ ...refundRequests ];
```

Comment:
If we're looking to deep-copy a variable, should we use `deepCopy`?
```suggestion
    return deepCopy(refundRequests);
```

Reply:
what's the difference? I think using native TS syntax might be better here

Reply:
Sorry for the delay on this - I wasn't re-requested / notified of this. The `deepCopy` is going to deep copy the `refundRequest` elements whereas `[...refundRequests]` are just a shallow copy of the elements in a new array instance.